fabric-creative-tab-api-v1/src/client/java/net/fabricmc/fabric/api/client/creativemodetab/v1/FabricCreativeModeInventoryScreen.java

```diff
@@ -69,21 +69,21 @@ default int getPageCount() {
 	}
 
 	/**
-	 * Returns an ordered list containing the item groups on the requested page.
+	 * Returns an ordered list containing the creative mode tabs on the requested page.
 	 */
-	default List<CreativeModeTab> getItemGroupsOnPage(int page) {
+	default List<CreativeModeTab> getCreativeModeTabsOnPage(int page) {
```

Comment:
The new names of these methods sound quite verbose. `getTabsOnPage` etc. are clear enough IMO without being as wordy.